examples/seismic/model.py

```diff
@@ -488,24 +502,41 @@ def domain_size(self):
 
 
 class Model(GenericModel):
-    """The physical model used in seismic inversion processes.
-
-    :param origin: Origin of the model in m as a tuple in (x,y,z) order
-    :param spacing: Grid size in m as a Tuple in (x,y,z) order
-    :param shape: Number of grid points size in (x,y,z) order
-    :param space_order: Order of the spatial stencil discretisation
-    :param vp: Velocity in km/s
-    :param nbpml: The number of PML layers for boundary damping
-    :param epsilon: Thomsen epsilon parameter (0<epsilon<1)
-    :param delta: Thomsen delta parameter (0<delta<1), delta<epsilon
-    :param theta: Tilt angle in radian
-    :param phi: Asymuth angle in radian
-
-    The :class:`Model` provides two symbolic data objects for the
+    """
+    The physical model used in seismic inversion processes.
+
+    Parameters
+    ----------
+    origin :
+        Origin of the model in m as a tuple in (x,y,z) order.
+    spacing : tuple of int
+        Grid size in m as a Tuple in (x,y,z) order.
+    shape : int
+        Number of grid points size in (x,y,z) order.
+    space_order : int
+        Order of the spatial stencil discretisation.
+    vp : float
+        Velocity in km/s
+    nbpml : int, optional
+        The number of PML layers for boundary damping.
+    dtype :
+
+    epsilon : float, optional
+        Thomsen epsilon parameter (0<epsilon<1).
+    delta : float
+        Thomsen delta parameter (0<delta<1), delta<epsilon.
+    theta : float
+        Tilt angle in radian.
+    phi : float
+        Asymuth angle in radian.
+
+    The `Model` provides two symbolic data objects for the
     creation of seismic wave propagation operators:
 
-    :param m: The square slowness of the wave
-    :param damp: The damping field for absorbing boundarycondition
+    m :
+        The square slowness of the wave.
+    damp :
+        The damping field for absorbing boundarycondition.
```

Comment:
"for absorbing boundary condition" or simply "for absorbing boundaries" @mloubout

Reply:
yes

Reply:
@georgebisbas reminder to update this

Reply:
thanks, fixed